diff --git a/Toolchains/Toolchain-Windows-x86_64.cmake b/Toolchains/Toolchain-Windows-x86_64.cmake
index 270178c..9c46ce8 100644
--- a/Toolchains/Toolchain-Windows-x86_64.cmake
+++ b/Toolchains/Toolchain-Windows-x86_64.cmake
@@ -1,3 +1,10 @@
+set(LLVM_NATIVE_TOOLCHAIN "/usr")
+set(UCRTVersion 10.0.16299.0)
+set(VCToolsVersion 14.11.25503)
+set(WinSDKRoot "/mnt/share/WinSDK")
+set(LLVM_MONOREPO_CHECKOUT "/root/llvm-project")
+set(APPLE_SWIFT_CHECKOUT "/root/llvm-win")
+
 set(UniversalCRTSdkDir "${WinSDKRoot}/Windows/10/SDK")
 set(VCToolsInstallDir "${WinSDKRoot}/Windows/10/MSVC/${VCToolsVersion}")
 
